README.md

```diff
@@ -200,9 +200,9 @@ Tokens can be used to populate any account Billing Info data through our API. Si
 
 **These endpoints accept tokens within billing info.**
 
+* Purchase [`create`](https://dev.recurly.com/docs/create-purchase)
 * Subscription [`create`](https://dev.recurly.com/docs/create-subscription)
 * Account [`create`](https://dev.recurly.com/docs/create-an-account), [`update`](https://dev.recurly.com/docs/update-account)
 * Billing Info [`update`](https://dev.recurly.com/docs/update-an-accounts-billing-info-token)
```

Comment:
Thoughts on referncing v3 here? For example: https://developers.recurly.com/api/v2019-10-10/index.html#operation/update_billing_info

Reply:
purchase create: https://developers.recurly.com/api/v2019-10-10/index.html#operation/create_purchase
subscription create: https://developers.recurly.com/api/v2019-10-10/index.html#operation/create_subscription
account create: https://developers.recurly.com/api/v2019-10-10/index.html#operation/create_account
account update: https://developers.recurly.com/api/v2019-10-10/index.html#operation/update_account
billing info update: https://developers.recurly.com/api/v2019-10-10/index.html#operation/update_billing_info